Add more explanations for custom domain example for serverless (#26072)
openshift/openshift-docs: modules/serverless-service-mesh-resources.adoc

----
$ oc apply -n istio-system -f <filename>
----

[IMPORTANT]
====

> {product-title} route must be created in the same namespace with your ServiceMeshControlPlane. In this example, ServiceMeshControlPlane is deployed in `istio-system` namespace.

Reviewer: ```suggestion
The {product-title} route must be created in the same namespace as the ServiceMeshControlPlane. In this example, the ServiceMeshControlPlane is deployed in the `istio-system` namespace.
```

Reviewer: @nak3 lmk whenever this is fixed and I'll ping the docs folks for review and merge. Thanks!

Author: Thank you @abrennan89 .  Updated!
